ledger/sandbox/src/main/resources/db/migration/h2database/V9__Package_upload_entries.sql

```diff
@@ -0,0 +1,35 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V9: package_upload_entries
+--
+-- This schema version adds a table for tracking DAML-LF package submissions
+-- It includes id to track the package submission and status
+---------------------------------------------------------------------------------------------------
+
+CREATE TABLE package_upload_entries
+(
+    ledger_offset    bigint not null,
+    recorded_at      timestamp          not null, --with timezone
+    -- SubmissionId for package to be uploaded
+    submission_id    varchar primary key not null,
+    -- participant id that initiated the package upload
+    participant_id   varchar            not null,
+    -- The type of entry, one of 'accept' or 'reject'
+    typ              varchar            not null,
```

Comment:
`entry_type`?

Reply:
same comment as above -- staying close to column naming for ledger_entries and configuration_entries